libraries/ms-drivers/inc/bts_7200_current_sense.h

```diff
@@ -4,49 +4,104 @@
 // Requires GPIO, interrupts, soft timers, and ADC to be initialized in ADC_MODE_SINGLE.
 // If using with MCP23008, requires I2C to be initialized.
 
+// Due to the fault handling procedures for the BTS7200, all BTS7200 pins should
+// only be manipulated through this driver.
+
 #include "adc.h"
 #include "gpio.h"
 #include "pca9539r_gpio_expander.h"
 #include "soft_timer.h"
 
+// Upper maximum for the possible leakage voltage that may be read from the SENSE pin at
+// T(env) < 80 C (see p.g. 27 of BTS7200 datasheet)
+#define BTS7200_MAX_LEAKAGE_VOLTAGE_MV 2
+
+// Nominal scaling factor k(ILIS) for current output at the SENSE pin in normal operation.
+#define BTS7200_IS_SCALING_NOMINAL 670
+
+// Voltage at the SENSE pin is limited to a max of 3.3V by a diode.
+// Due to to this function, since any fault current will be at least 4.4 mA (see p.g. 49)
+// the resulting voltage will be 4.4 mA * 1.6 kOhm = ~7 V. Due to this,
+// voltages approaching 3.3V represent a fault, and should be treated as such.
+#define BTS7200_MAX_VALID_SENSE_VOLTAGE_MV 3200
```

Comment:
Hm, this looks like it's specific to the power distro board, not generic to every BTS7200? If so it'd be better to make this configurable in the settings so it can vary in future boards. (you could put a "suggested value" in a comment though with these justifications if that makes sense)